core/src/ot_config.c

```diff
@@ -233,7 +234,8 @@ int ot_config_stop(void)
 bool ot_config_is_ready(void)
 {
 	/* Device is ready when its role is set to Child */
```

Comment:
Change the comment.

Reply:
Now it can be ready as router and leader too

Reply:
Done